src/components/Input/DropDown/DropDownGroup.styles.js

```diff
@@ -219,3 +219,15 @@ export const StyledKeyboardProvider = styled(KeyBoardProvider)`
     opacity: 1;
   }
 `;
+
+export const StyledOuterLabel = styled.span`
```

Comment:
Would using the `label` element be more appropriate here?:

Reply:
Oh, sure) Didn't even notice this, thanks for mentioning!